miden-crypto/src/lib.rs

```diff
@@ -30,36 +30,37 @@ pub use word::{Word, WordError};
 
 /// An alias for a key-value map.
 ///
-/// By default, this is an alias for the [`alloc::collections::BTreeMap`], however, when the
-/// `hashmaps` feature is enabled, this is an alias for the `hashbrown`'s `HashMap`.
-#[cfg(feature = "hashmaps")]
-pub type Map<K, V> = hashbrown::HashMap<K, V>;
+/// By default, this is an alias for the [`alloc::collections::BTreeMap`]; when the `std` feature is
+/// enabled, this is an alias for the `std::collections::HashMap` (iteration order is not
+/// deterministic in that case).
+#[cfg(feature = "std")]
+pub type Map<K, V> = std::collections::HashMap<K, V>;
```

Comment:
I think the comment is not exactly correct here: by default, we enable `std::collections::HashMap`.